### PR TITLE
Updated help text for Time zone

### PR DESCRIPTION
**Pull request by egc112**
Updated help text for Time zone, users often set Time servers by url which is not always resolved (depending on configuration) or choose a non working time server.

### CODE REVIEW

File: src/router/kromo/dd-wrt/lang_pack/english.js [hunk 1]
```diff
@@ -1250,7 +1250,7 @@ hidx.right10="This is the subnet mask of the router.";
 hidx.right12="Allows the router to manage your IP addresses.";
 hidx.right14="The address you would like to start with.";
 hidx.right16="You may limit the number of addresses your router hands out. 0 means only predefined static leases will be handed out.";
-hidx.right18="Choose the time you are in an summer time (DST) period. The router can use local time or UTC time.";
+hidx.right18="Choose the time zone you are in. Leave Server IP/Name blank so that built-in IP addresses of time servers are used (recommended)";
```
Comment (the-j0k3r) [inline src/router/kromo/dd-wrt/lang_pack/english.js:1253]:
```suggestion
hidx.right18="Choose the time zone you are in. Leave the Server IP/Name field blank so the built-in IP addresses of the time server is used instead (recommended)";
```

**Comment by egc112** [comment]
Anything which encourages the user to leave the field blank is fine with me :)

**Comment by the-j0k3r** [comment]
I was looking at the grammar side a little, but feel free to use my suggestion or not, I cant seem to be able to do a contrib to your fork/branch for this commit over browser.

**Comment by the-j0k3r** [comment]
wait, this wasn't merged, what changed?

**Comment by egc112** [comment]
I emailed him with the patch will be merged tomorrow :)

**Comment by the-j0k3r** [comment]
tbh I would hate to be brainslayer and his email inbox probably needs a full time attendant =) lhahahaha

**Comment by egc112** [comment]
I use a different email address which seems less crowded

**Pull request closed without merging**